requirements.txt

```diff
@@ -1,13 +1,13 @@
 # This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
+# on Mozilla Web QA projects
 
-PyYAML==3.10
-execnet==1.0.9
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.26
-pytest==2.7.0
+PyYAML==3.11
+execnet==1.4.1
+certifi==2016.2.28
+chardet==2.3.0
+py==1.4.31
```

Comment:
Remove all downstream dependencies. I believe PyYAML, execnet, certifi, chardet, and py are all downstream dependecies. We let pytest determine its dependencies now.